get-new-tool-versions/parsers/xamarin-parser.psm1

```diff
@@ -0,0 +1,30 @@
+using module "./base-parser.psm1"
+
+class XamarinVersionsParser: BaseVersionsParser {
+    [PSCustomObject] GetAvailableVersions() {
+        $allVersions = $this.ParseAllAvailableVersions()
+        return $allVersions
+    }
+
+    [hashtable] GetUploadedVersions() {
+        $url = $this.BuildGitHubFileUrl("actions", "virtual-environments", "main", "images/macos/toolsets/toolset-11.0.json")
```

Comment:
Is it enough to check only 11.0 toolset?

Reply:
I think it's enough to check only macOS 11.0 which contains the latest changes.